docs/components/popover.md

```diff
@@ -420,6 +455,57 @@ This is the slot that takes in the content of the popover.
 </KPop>
 ```
 
+- `footer`
+This is an optional slot that takes in content for the footer bar.
+
+```vue
+<KPop title="Cool header">
+  <!-- Your element goes here -->
```

Comment:
This is pretty general, should we be more explicit about what element goes here?

Reply:
Updated slot description text